lib/plugins/aws/index.js

```diff
@@ -33,7 +33,7 @@ class SDK {
 
   request(service, method, params, stage, region) {
```

Comment:
yup, but this is a separate task as lots of files would have to be changed for that.